exercises/.gitignore

```diff
@@ -1,3 +1,7 @@
 *.out
 *.exe
+<<<<<<< HEAD
 
+=======
+.vscode/
+>>>>>>> origin
```

Comment:
merge conflict https://www.simplilearn.com/tutorials/git-tutorial/merge-conflicts-in-git#:~:text=A%20merge%20conflict%20is%20an,on%20different%20lines%20or%20branches.

Spero di fare qualche esempio durante gli incontri

Reply:
@Helias non capisco se ho sistemato o ci sta ancora il merge conflict...